example-article.json

```diff
@@ -76,6 +104,35 @@ This file specifies what posts look like:
     ]
 }
 
+
+# Example 
+# http://service/posts/{post_id}/comments access to the comments in a post
+# http://service/posts/{post_id}/comments?page=4
+# http://service/posts/{post_id}/comments?page=4&size=40
+{
+	"query": "comments",
```

Comment:
I added the "query" key to make it consistent with the other queries. If they are not required for anything I don't see their use in any of them. (Our team not using it for anything but some other team may be)